Add quickstart example for task queue functions. (#971)
firebase/functions-samples: quickstarts/taskqueues-backup-images/README.md

> # Quickstart: Back up images using a Task Queue Function

Reviewer: Do you think the README should discuss the values chosen for rate limits, backoff, etc?  

Not necessary afaik, but I feel like I'd like to discuss them in the doc/guide.

Author: Great suggestion! I'll try to expand on the task queue configuration on the README.md.

This quickstart demonstrates how to setup a Task Queue Function using the Firebase SDK for Cloud Functions.

## Introduction

Task queue functions make it easy to manage the execution, dispatch, and delivery of a large number of distributed tasks.

We leverage its power here to set up a service that backs up all images from NASA's [Astronomy Picture of the Day](https://apod.nasa.gov/apod/astropix.html).

> Task queue functions are powered by [Google Cloud Tasks](https://cloud.google.com/tasks). Learn more about the task queue functions at TODO.

Author: Yep that's what I'm hoping to include here. Do we know roughly where that would be?

# Quickstart: Back up images using a Task Queue Function
This quickstart demonstrates how to setup a Task Queue Function using the Firebase SDK for Cloud Functions.

## Introduction

Task queue functions make it easy to manage the execution, dispatch, and delivery of a large number of distributed tasks.

We leverage its power here to set up a service that backs up all images from NASA's [Astronomy Picture of the Day](https://apod.nasa.gov/apod/astropix.html).

> Task queue functions are powered by [Google Cloud Tasks](https://cloud.google.com/tasks). Learn more about the task queue functions at TODO.

Reviewer: Suggest our Firebase-ish tutorial, which will live at https://firebase.google.com/docs/functions/beta/task-functions

Nothing there now, but once this sample is tagged, a doc CL will take flight based on the very complete Gdoc draft.

> Task queue functions are powered by [Google Cloud Tasks](https://cloud.google.com/tasks). Learn more about the task queue functions at TODO.

Reviewer: Sorry, for v1 that will be https://firebase.google.com/docs/functions/task-functions

## Functions
The sample code consists of 2 functions:

### 1. `backupApod`
A task queue function responsible for processing the logic for backing up the Astronomy Picture of the Day ("apod") for the given date. This function will be triggered for every task enqueued on the corresponding queue created in Cloud Tasks.

> Task queue functions come with a powerful set of configuration to precisely control rate limits and retry behavior of a task queue. Learn more about configuring task queue functions at TODO.

Reviewer 2: ```suggestion
Task queue functions come with a powerful set of configuration to precisely control rate limits and retry behavior of a task queue. [See the documentation](https://cloud.google.com/tasks/docs/creating-queues) to learn more about configuring task queue functions.
```
